frontend/javascripts/oxalis/view/components/setting_input_views.tsx

```diff
@@ -9,6 +9,8 @@ import messages from "messages";
 
 const ROW_GUTTER = 1;
 
+const ANTD_TOTAL_SPAN = 24;
```

Comment:
I added these changes because I added a setting for the "smooth mesh" behavior at first. However, I removed the setting again. The changes here are quite generic, though, and might be useful later which is why I left them here.